brd-ios/breadwallet/SignInInteractor.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Created by Equaleyes Solutions Ltd
+//
+
+import UIKit
+
+protocol SignInBusinessLogic {
+    // MARK: Business logic functions
+    
+    func login(request: SignIn.LoginData.Request)
+}
+
+protocol SignInDataStore {
+    // MARK: Data store
```

Comment:
should the data store not have email / password properties?

Reply:
already done. will push.